### ISSUE TITLE
Objects aren't appearing in the editor on Windows and Mac

### ISSUE THREAD

**Issue by SomeWeirdo** [4ian/GDevelop#851]
## Describe the bug
Certain objects won't appear in the editor but appear when you click to play the game, I've linked the game below.

<a href="https://gifyu.com/image/w7da"><img src="https://s2.gifyu.com/images/ezgif.com-resize658a0e41d5065726.gif" alt="ezgif.com-resize658a0e41d5065726.gif" border="0" /></a>

## To Reproduce
Can't, it's random. Here's a link to where it's happening https://ufile.io/cgc7q (This is my personal project, I've deleted all the events because I don't want it stolen).

Basically, the "slot" object(s) should be on the screen, they don't appear in the editor but appear when you click the play button.

## Other details
OS X 10.13.4 on the latest version of GDevelop. I was working on this with someone else who was on Windows who was having the same problem.

**Comment by zatsme** [comment]
Yes, same, happy to try to debug next time it happens, is there any way we can provide relevant logging?

**Comment by 4ian** [comment]
It's hard to debug as seems to be random and hard to see what kind of information would be useful. I wonder if it's some kind of memory corruption or something fishy in the way iteration on the instances to render is done.

**Comment by Bouh** [comment]
Close because link to files on this report are dead.
If a new issue with a similar problem come will be able to check whats wrong.

I hope it fixed with the multiples updates of PixiJS over time.
Note it can be related to https://github.com/4ian/GDevelop/issues/2212
